web_src/js/features/common-global.js

```diff
@@ -142,6 +142,15 @@ export function initGlobalCommon() {
       window.location = href;
     }
   });
+
+  // loading-button this logic used to prevent push one form more than one time
+  $(document).on('click', '.button.loading-button', function () {
+    const $btn = $(this);
+
+    if ($btn.hasClass('loading')) return false;
```

Comment:
Does return false prevent it to bubble event up? I think it still needs e.preventDefault.

Reply:
Actually, `return false` in jQuery means preventDefault and stoppropagation at the same time.

Reply:
I don't know, that's why I asked 😅 have not really developed in jQuery for ages 🤣